app/page/send/index.tsx

```diff
@@ -0,0 +1,210 @@
+import * as ethers from "ethers"
+import { useCallback, useContext, useState, type ChangeEvent } from "react"
+import { Link } from "wouter"
+
+import { StepBackHeader } from "~app/component/stepBackHeader"
+import { ProviderContext } from "~app/context/provider"
+import { Path } from "~app/path"
+import { useAccount, useTokens } from "~app/storage"
+import { getChainName, getErc20Contract } from "~packages/network/util"
+import address from "~packages/util/address"
+import number from "~packages/util/number"
+import { type Token } from "~storage/local/state"
+import type { BigNumberish, HexString } from "~typing"
+
+const SelectToken = ({ tokens, onSelectToken }) => {
+  const handleAssetChange = async (event: ChangeEvent<HTMLSelectElement>) => {
+    const tokenAddress = event.target.value
+    console.log(`tokenAddress: ${tokenAddress}`)
+    const token = tokens.find((token) => token.address === tokenAddress)
+    onSelectToken(token)
+  }
+
+  return (
+    <div className="flex">
+      <label className="col-span-1">Asset:</label>
+      <select
+        id="asset"
+        value={tokens[0].address}
```

Comment:
It would be better to import the token state so that it can get the latest token state.

```typescript
const SelectToken = ({ tokens, token, onSelectToken }) => {
// ...
  return (
// ...
      <select
        id="asset"
        value={token.address}
// ...
```

Or use the local state:

```typescript
const SelectToken = ({ tokens, onSelectToken }) => {
  const [token, setToken] = useState<Token>(tokens[0])
  const handleAssetChange = async (event: ChangeEvent<HTMLSelectElement>) => {
// ...
    setToken(token)
// ...
  return (
// ...
      <select
        id="asset"
        value={token.address}
// ...
  }
```

Reply:
Thanks for your comment. :)
In this PR, the main task is to break down each step, so the `SelectToken` part is in another PR #170

Reply:
I understand now 0.0., thank you!!